gwos/contracts/delegate-cell-lock/src/entry.rs

```diff
@@ -0,0 +1,44 @@
+use ckb_std::{
+    ckb_constants::Source,
+    high_level::{
+        load_cell_data, load_cell_lock_hash, load_cell_type_hash, load_script, load_witness_args,
+        QueryIter,
+    },
+};
+
+use crate::error::Error;
+
+/// Lock args: type hash (and usually type-id) of delegate-cell.
```

Comment:
We should explain the `delegate-cell` (admin cell) and the `key-cell` (block producer's cell) with an actual lock script to help reviewers understand this script.

* `delegate-cell` plays the role of administrator. We assume it is controlled by the [secp256k1 multisig lock](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0024-ckb-genesis-script-list/0024-ckb-genesis-script-list.md#secp256k1multisig).
* `key-cell` plays the role of the singleton block producer. We assume it is controlled by the [secp256k1 blake160 lock](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0024-ckb-genesis-script-list/0024-ckb-genesis-script-list.md#secp256k1blake160).

Please add the explanation in the comment.

Reply:
Added.